### ISSUE TITLE
How to make evaluation

### ISSUE THREAD

**Issue by 0215Arthur** [siyongxu/GNUD#6]
I'm very interested in your work. After reading your paper, I’d like to learn more about the experiment setting, especially about the evaluation. 
In the experiment, the Classification metrics are selected, so how to select the negative samples?
Randomly sampling? what's the ratio of positive to negative samples?
could you share the evaluation setting?

Thank you very much.

**Comment by siyongxu** [comment]
Yes,we randomly sample the negative samples from the first six days that the target user never click for training data and from the whole week for test data. The ratio is simply set to 1:1.

**Comment by XinyuanLu00** [comment]
Hi authors, I'm very interested in your work. Is there a code available for your feature extraction process,.i.e. the process of generating Data.npz?
